### PR TITLE
Feature/1780/support elasticsearch6

### PR DESCRIPTION
**Pull request by nrzhao4**
- changes for updating to elasticsearch 6.8+ and corresponding to changes proposed in [3899](https://github.com/dockstore/dockstore/pull/3899)
- results from search come from two indices rather than one, handle them by the `index` property rather than `_type`
- slight query change for the autocomplete terms

### CODE REVIEW

**Review by denis-yuen — COMMENTED** [review]
On tests passing, would merge group together after storytime/demo

**Review by wshands — APPROVED** [review]

**Review by NatalieEO — APPROVED** [review]

**Review by denis-yuen — APPROVED** [review]
Approved for merge after webservice counterpart is merged

**Review by garyluu — APPROVED** [review]

**Pull request merged**
